### ISSUE TITLE
Init git-flow repo for non-default configuration

### ISSUE THREAD

**Issue by leomperes** [OpherV/gitflow4idea#147]
Getting this error when choosing production and next releases branches:

```
14:51:26.317: [live-portal] git -c core.quotepath=false flow init
Which branch should be used for bringing forth production releases?
   - master
   - staging
14:51:26.654: master
Branch name for production releases: [master] 
Which branch should be used for integration of the "next release"?
   - staging
14:51:26.767: staging
Production and integration branches should differ.
Branch name for "next release" development: [master] 
```

![screenshot 2017-06-15 15 00 06](https://user-images.githubusercontent.com/1631712/27197242-79b734c0-51db-11e7-9c27-afad7aa0de5a.png)

**Comment by OpherV** [comment]
I actually already sorted this out in cbc70bb3a7f448257ea9e80c0b5d6155f21267db- the fix will be available in the upcoming version

**Comment by leomperes** [comment]
Thanks @OpherV , will be waiting!

**Comment by OpherV** [comment]
Available in 0.6.1

**Comment by leomperes** [comment]
Updated, thanks!
